ci/checks/enforce-docs-up-to-date.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+pip install bs4
+pip install docutils
+pip install pygments
+#pip install ../../
+
+pwd
+ls
+previous_pwd=$(pwd)
+#cd ../../
+
+current_docs=$(cat git_machete/docs.py)
+generated_docs=$(python docs/generate_docs.py)
+if [[ $current_docs != "$generated_docs" ]]; then
+  cd $previous_pwd || return
```

Comment:
Hmm are the `cd`s needed? Current directory is confined to a single bash process, and in `ci/checks/run-all-checks.sh` we execute each script (including this one) in a separate shell process

Reply:
Yes, you are right, the 'cd's are indeed unnecessary